app/screens/home/channel_list/plus_menu/index.tsx

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {View} from 'react-native';
+
+import BottomSheetContent from '@screens/bottom_sheet/content';
+
+import PlusMenuItem from './menu_item';
+
+const PlusMenuList = () => {
+    return (
+        <BottomSheetContent
+            showTitle={false}
+            showButton={false}
+        >
+            <View>
+                <PlusMenuItem
```

Comment:
There is already a "SlideUpPanelItem" component. Can you check if that would work for you here, instead of the "PlusMenuItem"?

Reply:
@larkox Yes, it's working but the font their is different